docs/getting-started/threat-intel-integrations.asciidoc

```diff
@@ -63,6 +63,6 @@ NOTE: For more information about enabling available threat intelligence filesets
 +
 NOTE: Threat intelligence indices aren’t required to be ECS compatible. However, we strongly recommend compatibility if you’d like your alerts to be enriched with relevant threat indicator information. You can find a list of ECS-compliant threat intelligence fields at {ecs-ref}/ecs-threat.html[Threat Fields].
 
-. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
+. Return to the Threat Intelligence view on the Overview page (*Dashboards -> Overview*). Refresh the page if indicator data isn't displaying.
```

Comment:
```suggestion
. Return to the Threat Intelligence view on the Overview dashboard (*Dashboards -> Overview*). Refresh the page if indicator data isn't displaying.
```

Reply:
Ah, I was wondering about this. Thanks, will incorporate it now.